packages/extension-browser/src/devtools/views/controls/external-doc.css

```
@@ -0,0 +1,7 @@
+.header {
+    margin-bottom: 0;
+}
+
+.list {
+    margin-top: .4rem 0 0;
```

Nit: `.4rem` is ~6.4px. Would `.5rem` (8px) work here so we don't have fractional values? (also can we include the equivalent `px` value in a comment like we do in other files?)
```suggestion
    margin-top: .4rem 0 0;
```

---

In fact, that property is wrong hehehe. margin top can only have one value.

---

Good catch, I completely missed that (kept reading this as `margin` instead of `margin-top`😅)